### DISCUSSION TITLE
Discord Unban Appeal

### DISCUSSION THREAD

**Discussion by DARSHGOYALMAGMA** [pterodactyl/panel#5172] [Support] [answered]
Hi i m Darsh and i put my report earlier also but not replied so i will tell my reason again. I was banned from pterodactyl For saying that i watched One of The Mods Youtube Video to make Pterodactyl Panel On Oracle Cloud VPS and pinged That Mod and After 10 - 20 Minutes i didnt even get warned and got banned and didnt even got DMs of Valkyrja Bot and No support i got. Please Unban Me I will Not do That Crime again.

Discord ID: 1192421981665108054
User Name: darsh261_not

**Comment by rowansc1** [comment, marked as answer]
Hiya, 

You only got a temp ban. You're due to be unbanned on the 2024-08-26_10:18:07 UTC.

Feel free to join back after that :)
